src/psycopt2d/utils.py

```diff
@@ -52,7 +52,7 @@ def drop_records_if_datediff_days_smaller_than(
         t2_col_name (str): _description_
         t1_col_name (str): _description_
         threshold_days (Union[float, int]): _description_
```

Comment:
Missing docstrings. I am not completely sure if this is correct.

```suggestion
        t2_col_name (str): Column name of a time column.
        t1_col_name (str): Column name of a time column.
        threshold_days (Union[float, int]): Threshold days.
```